sui-execution/src/lib.rs

```diff
@@ -21,7 +21,7 @@ mod v0;
 #[cfg(test)]
 mod tests;
 
-pub const SUIVM: u64 = u64::MAX;
+pub const SUIVM: u64 = 2;
```

Comment:
this is strictly speaking not needed but to make things less confusing. 
And even with the `u64::MAX` the problem is there